deployments/puppet/spec/classes/init_spec.rb

```diff
@@ -9,7 +9,15 @@
   end
 
   on_supported_os.each do |os, facts|
-    let(:facts) { facts }
+    # When running rake spec it checks for absolute paths and on Windows paths
+    # are built from facts. However, the needed facts are not set since rake
+    # spec is running on a non-windows system.
+    let(:facts) {{
+      'win_temp' =>'C:\\Windows\\Temp',
+      'win_programfiles' => 'C:\\Program Files',
+      'win_programdata' => 'C:\\ProgramData',
+      'win_systemdrive' => 'C:'}.merge( facts )
```

Comment:
I'm concerned that these will break in a customer deployment. Is the added comment still relevant even in non-CI/CD environments? (Or is this logic even used in the deployment itself?)

_I'm not very familiar with this functionality, so just want to make sure._

Reply:
This is the `spec` for the rake test. On actual deployments the facts are created as expected, i.e.: on Windows those facts are going to be present. Perhaps I need to update the comment to make that clear 🤔

Reply:
3a21cf9f